app/src/main/res/values/strings.xml

```diff
@@ -26,5 +26,6 @@
     <string name="scale_pref_summary">Scale for recording text</string>
     <string name="hello_world">Hello world!</string>
     <string name="noProject">No projects found. Please choose a project in HearThis Desktop and synchronize it using the Sync button.</string>
+    <string name="no_use_without_record">This program is not much use without audio recording!</string>
```

Comment:
I think it will be important to (re-)request access when they try to record, if possible. That way, it will be perfectly obvious why HT wants access to their microphone. It's possible that we don't even need to request access up-front at all. Until they try to record, we don't need it. If we do want/need to request permission up-front, would this toast wording be better? Open Settings to give HearThis access to the microphone so you can use it to record audio.

Reply:
Done